web/docs/metrics.md

```diff
@@ -1,106 +1,43 @@
 # Metrics
 
-Tenzir collects various metrics during execution. This reference describes what
-metrics are available and what they mean. We describe how to [collect
-metrics](setup-guides/collect-metrics.md) in the corresponding user guide.
+:::warning Experimental
+This flag and the schema of the returned events are subject to change.
+:::
 
-For the reference table below, these symbols indicate presense of additional
-metadata:
+## `tenzir.metrics.operator`
 
-|Symbol|Key|Value|
-|-:|-|-|
-|🔎|`query`|A UUID to identify the query.|
-|🪪|`issuer`|A human-readable identifier of the query issuer.|
-|💽|`partition-type`|One of "active" or "passive".|
-|#️⃣|`partition-version`|The internal partition version.|
-|💾|`store-type`|One of "parquet" or "feather".|
-|🗂️|`schema`|The schema name.|
+```
+type tenzir.metrics.operator = {
+  /// The ID of the pipeline where the associated operator is from.s
+  pipeline_id: string,
+  /// The ID of the operator inside the pipeline referenced above.
+  operator_id: string,
+  source: bool,
+  transformation: bool,
+  sink: bool,
+  /// The time when this event was emitted, which is immediately after the
+  /// collection period (not before).
+  timestamp: time,
+  /// True if the associated pipeline was created by running an open pipeline in
+  /// the explorer. If false, then the pipeline has a name and is visible in the
+  /// overview.
+  hidden: bool,
+  /// Measurement of the incoming stream (see below).
+  input: tenzir.metrics.operator.measurement,
+  /// Measurement of the outgoing stream (see below).
+  output: tenzir.metrics.operator.measurement,
+}
+```
```

Comment:
Did you try putting these in a table?

|key|type|description|
|-|-|-|
|`pipeline_id`|`string`|The ID of the pipeline where the associated operator is from.|
|...|...|...|
|`output.unit`|`string`|The type of the output elements; either `void`, `bytes`, or `events`|
|`output.elements`|`uint64`|The number of bytes or events that were seen during the collection period. Non-cumulative.|
|`output.approx_bytes`|`uint64`|If `output.unit` is `events`, then this is an estimate of the memory footprint of the associated events. Otherwise, this is the same as `output.elements`.|